src/main/scala/nl.knaw.dans.easy.validatebag/rules/ProfileVersion0.scala

```diff
@@ -58,6 +58,7 @@ object ProfileVersion0 {
     NumberedRule("2.2(b)", containsFile(Paths.get("metadata/files.xml")), dependsOn = Some("2.1")),
     // 2.3 does not state restrictions, so it does not need checking
     NumberedRule("2.5", containsNothingElseThan(Paths.get("metadata"), Seq("dataset.xml", "files.xml", "agreements.xml")), dependsOn = Some("2.1")),
+    NumberedRule("2.6", containsDir(Paths.get("data"))),
```

Comment:
[According to the specs](https://tools.ietf.org/html/draft-kunze-bagit#section-2.1.2) a bag must have a payload directory. In other words: it cannot be valid (or virtually-valid) without one. So this requirement is implied by [rule 1.1.](https://github.com/DANS-KNAW/easy-specs/blob/master/dans-bagit-profile/v0/dans-bagit-profile-v0.md#11-validity).

Reply:
Changed dependency to 1.1.1 and deleted the newly added rule 2.6